declarative-schedule-doc.md

```diff
@@ -177,7 +177,7 @@ test_data:
 ```
 ...
 test_data:
-  - !include: 
```

Comment:
can I kindly ask you to test this scenario with some VR, just in case? I think it should work with both, but your fix is correct, it is just ot be sure that is parsed properly that way.

Reply:
It's a doc, not affecting any scenario, In all yamls we use it as a key, not as a list item.

Reply:
I already tried both scenarios, the pre-existing one didn't even run , after the fix it works.
before: http://falafel.suse.cz/tests/425
after: http://falafel.suse.cz/tests/426/file/autoinst-log.txt